NERODevelopment/src/controllers/offviewmodel.cpp

```diff
@@ -0,0 +1,13 @@
+#include "OffViewModel.h"
+
+OffViewModel::OffViewModel(QObject *parent) : QObject(parent)
```

Comment:
no this file shouldnt exist, you should add getters to the current model for all of these values and add them to the current macro file for the model